src/plugins/obj/obj_backend.cpp

```
@@ -16,40 +16,35 @@
 // Obj Engine Implementation
 // -----------------------------------------------------------------------------
 
-// TODO: Consider a registration pattern as more vendor engines are added.
+namespace {
+
+std::string
+getAccelType(nixl_b_params_t *custom_params) {
+    if (!custom_params) return "";
+    auto it = custom_params->find("type");
+    return (it != custom_params->end()) ? it->second : "";
+}
```

Cppcheck (2.20.0)

[style] The function 'waitUntilIdle' is never used.

(unusedFunction)